src/views/scans/showScansModal.tsx

```diff
@@ -50,39 +49,12 @@ const ShowScansModal: React.FC<ShowScansModalProps> = ({
     columnIndex
   });
 
-  /* ToDo: This code is for future use around Discovery-437. If it isn't used when the issue has resolved remove it
- *
-  const isJobSelected = (job: ScanJobType) => { return selectedJobs.includes(job.id); };
-  const isJobSelectable = (job: ScanJobType) => { return true || job.status === "completed"; };
-  const selectableJobs = scanJobs ? scanJobs.filter(isJobSelectable) : [];
-  const onSelectJob = (job: ScanJobType, rowIndex: number, isSelecting: boolean) => {
-      setSelectedJobs((prevSelected) => {
-          const otherSelected = prevSelected.filter((j) => j !== job.id);
-          return isSelecting && isJobSelectable(job) ? [...otherSelected, job.id] : otherSelected;
-        });
-  };
-  const areAllJobsSelected = false;
-
-  const selectAllJobs = (isSelecting: boolean = true) => {
-      setSelectedJobs(isSelecting ? selectableJobs.map((j) => j.id) : []);
-  };
-  const getSortableRowValues = (scanJob: ScanJobType): (Date | string)[] => {
-  const { end_time, status } = scanJob;
-  return [end_time, status];
-};
-
-  let sortedJobs = scanJobs;
-  if (activeSortIndex !== undefined && scanJobs) {
-    sortedJobs = scanJobs.sort((a, b) => {
-      const aValue = getSortableRowValues(a)[activeSortIndex];
-      const bValue = getSortableRowValues(b)[activeSortIndex];
-      console.log({ aValue, bValue });
-      if (activeSortDirection === 'asc') {
-        return (aValue as string)?.localeCompare(bValue as string);
-      }
-      return (bValue as string)?.localeCompare(aValue as string);
-    });
-  }*/
+  // Sort scanJobs by scan time (latest first)
+  const sortedScanJobs = (scanJobs || []).sort((a, b) => {
+    const timeA = a.end_time ? new Date(a.end_time).getTime() : new Date(a.start_time).getTime();
+    const timeB = b.end_time ? new Date(b.end_time).getTime() : new Date(b.start_time).getTime();
+    return timeB - timeA; // Sort in descending order (latest first)
+  });
```

Comment:
optional: this entire helper can be made similar to how we handled the other inline where we know `start_time` exists

```
  const sortedScanJobs = (scanJobs || []).sort((a, b) => {
    const timeA = new Date(a.end_time || a.start_time).getTime();
    const timeB = new Date(b.end_time || b.start_time).getTime();
    return timeB - timeA; // Sort in descending order (latest first)
  });
```

future:
we should review the `scanJobs || []` as a default param instead